spec/features/user_cannot_access_other_user_info_spec.rb

```diff
@@ -1,6 +1,23 @@
+require 'rails_helper'
 
+describe 'logged in user cannot access other users info', type: :feature do
+  let(:user_1) { Fabricate(:user) }
+  let(:user_2) { user = User.new(first_name: "jabberowcky")
```

Comment:
Avoid using {...} for multi-line blocks.<br>Block body expression is on the same line as the block start.